backend/typescript/services/implementations/adminService.ts

```diff
@@ -47,6 +51,58 @@ class AdminService implements IAdminService {
     }
     return waiverDto;
   }
+
+  async updateFormTemplate(form: FormTemplateDTO): Promise<FormTemplateDTO> {
+    let formTemplateDTO: FormTemplateDTO | null;
+    try {
+      /* eslint no-underscore-dangle: 0 */
+      const formQuestions: Array<FormQuestionDTO> = [];
+      await Promise.all(
+        form.formQuestions.map(async (formQuestion, i) => {
+          const question = await formQuestionModel.create({
+            type: formQuestion.type,
+            question: formQuestion.question,
+            required: formQuestion.required,
+            description: formQuestion.description,
+            options: formQuestion.options,
+          });
+          formQuestions[i] = question._id;
+        }),
+      );
+      await formTemplateModel.remove({});
```

Comment:
ah i think we want to delete the questions in the actual formQuestions collection? so old unused questions aren't hanging around. so before creating the new question documents we should delete all current questions in formQuestions